modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -253,6 +253,27 @@ definition {
 		return ${response};
 	}
 
+	@summary = "Default summary"
+	macro createObjectRelatedToAnotherObject(en_US_plural_label = null, name = null, relatedObject = null, relatedObjectErc=null, relationship=null) {
+		Variables.assertDefined(parameterList = "${en_US_plural_label},${name},${relationship},${relatedObject},${relatedObjectErc}");
```

Comment:
${relatedObject} is relatedObjectName? If so, please update

Reply:
${relationship} is relationshipName? If so, please update

Reply:
Updated and rebased, thanks!